metrics/models.py

```diff
@@ -0,0 +1,24 @@
+from django.contrib.auth.models import User
+from django.db import models
+
+from topics.models import Article, Topic
+
+
+class Metric(models.Model):
+    article = models.ForeignKey(Article)
+    topic = models.ForeignKey(Topic)
+    owner = models.ForeignKey(User)
+
+    location = models.IntegerField()
```

Comment:
I thought of it as the index in the article. The article could change too anyway so anything we do will probably never be good enough